src/util/storybook-docs/ImportPath2.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from 'react';
+import { DocsContext } from '@storybook/addon-docs/blocks';
+import CloneIcon from 'lib/icons/react/CloneIcon';
+import CheckIcon from 'lib/icons/react/CheckIcon';
+import useToggleHover from './useToggleHover';
+import useClipboard from './useClipboard';
+
+export const ImportPath = () => {
+  const toggleHover = useToggleHover()[1];
+  const [copiedText, copyToClipboard] = useClipboard();
+  const context = useContext(DocsContext);
+
+  const componentName = context.kind.split('/')[1];
```

Comment:
I'm merely taking `components/Avatar` and just taking `Avatar`. We know the rest of the path.